mailpoet/lib/Automation/Integrations/MailPoet/Analytics/RegisterAnalytics.php

```diff
@@ -0,0 +1,26 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Automation\Integrations\MailPoet\Analytics;
+
+use MailPoet\API\REST\API;
+use MailPoet\Automation\Engine\Hooks;
+use MailPoet\Automation\Engine\WordPress;
+use MailPoet\Automation\Integrations\MailPoet\Analytics\Endpoints\OverviewEndpoint;
+
+class RegisterAnalytics {
```

Comment:
This sounds like a method name. Can we use just `Analytics`?

Reply:
dd87bb888